node/package.json

```diff
@@ -36,5 +36,8 @@
     "google-protobuf": "^3.21.2",
     "jsrsasign": "^10.6.1",
     "uuid": "^9.0.0"
-  }
+  },
+  "files": [
```

Comment:
We only needs to publish the files inside the `dist` folder because the other files are TypeScrpit files.